packages/Bio/src/utils/multiple-sequence-alignment-ui.ts

```diff
@@ -161,3 +180,21 @@ async function onColInputChange(
     _package.logger.error(errMsg);
   }
 }
+
+type MSADialogType = 'kalign' | 'pepsea';
+
+function switchDialog(
+  pepseaInputRootStyles: CSSStyleDeclaration[], kalignInputRootStyles: CSSStyleDeclaration[], dialogType: MSADialogType
+) {
+  if (dialogType === 'kalign') {
+    for (const inputRootStyle of pepseaInputRootStyles)
+      inputRootStyle.display = 'none';
```

Comment:
I see you're trying to change the style of every root element
firstly, change not the display value but just assign or unassign the class name
also, I can suggest you a better solution
maybe instead of changing the display of every element you better change the display value of the whole container, where these roots are stored?
and in this container you will refer to every root element using CSS `>` selector?